clients/da-vinci-client/src/main/java/com/linkedin/davinci/storage/DiskHealthCheckService.java

```diff
@@ -187,46 +219,61 @@ public void run() {
           // delete the temporary file at the end
           tmpFile.deleteOnExit();
 
-          try (PrintWriter printWriter = new PrintWriter(tmpFile, "UTF-8")) {
-            String message = String.valueOf(System.currentTimeMillis());
-            // write 64KB data to the temporary file first
-            int repeats = TMP_FILE_SIZE_IN_BYTES / message.length();
-            for (int i = 0; i < repeats; i++) {
-              printWriter.println(message);
-            }
-            printWriter.flush();
-
-            // Check data in it.
-            errorMessage = null;
-            boolean fileReadableAndCorrect = true;
-            try (BufferedReader br =
-                new BufferedReader(new InputStreamReader(new FileInputStream(tmpFile), StandardCharsets.UTF_8))) {
-              for (int i = 0; i < repeats; i++) {
-                String newLine = br.readLine();
-                if (!Objects.equals(newLine, message)) {
-                  errorMessage =
-                      "Content in health check file is different from what was written to it; expect message: "
-                          + message + "; actual content: " + newLine;
-                  fileReadableAndCorrect = false;
-                  break;
-                }
-              }
-            }
+          String message = String.valueOf(System.currentTimeMillis());
```

Comment:
If we use `Time` interface here, we won't need this: `forceOverrideMessage` since we can use `MockTime` in the unit test.

Reply:
I can give it a try